modules/search/pom.xml

```diff
@@ -131,7 +131,7 @@
     <dependency>
       <groupId>com.spatial4j</groupId>
       <artifactId>spatial4j</artifactId>
-      <version>0.4.1</version>
+      <version>0.4</version>
```

Comment:
Is there a special reason for the downgrade?

Reply:
Yes. This is changed so that the library versions of what is actually used and included through other libraries. Just compile the module. Maven will even print a warning about this.